lib/puppetserver/ca/puppetserver_config.rb

```diff
@@ -0,0 +1,84 @@
+require 'hocon'
+
+module Puppetserver
+  module Ca
+    # Provides an interface for querying Puppetserver settings w/o loading
+    # Puppetserver or any TK config service. Uses the ruby-hocon gem for parsing.
+    class PuppetserverConfig
+
+      def self.parse(config_path = nil)
+        instance = new(config_path)
+        instance.load
+
+        return instance
+      end
+
+      attr_reader :errors, :settings
+
+      def initialize(supplied_config_path = nil)
+        @using_default_location = !supplied_config_path
+        @config_path = supplied_config_path || "/etc/puppetlabs/puppetserver/conf.d/ca.conf"
```

Comment:
Is there any use case where puppetserver's conf file might be in a user-local directory, besides running from source? (I think for that case we should specify `--config` manually)